modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -22,20 +25,26 @@
 #   specific machines.
 #   Default: true
 #
+# [*version*]
+#   The version of elasticsearch to install.  This must specify an exact
+#   version (eg 1.4.2)
+#
 class govuk_elasticsearch (
   $version,
   $cluster_hosts = ['localhost'],
   $cluster_name = 'elasticsearch',
   $heap_size = '512m',
   $number_of_shards = '3',
   $number_of_replicas = '1',
-  $minimum_master_nodes = '1',
+  $minimum_master_nodes = '2',
```

Comment:
I think changing this to 2 is correct but could we do it in a separate commit? It's unrelated to the other changes going on here.

We could actually remove this parameter entirely because we only ever set it to the same value in each of the 3 places its called from.